pyproject.toml

```diff
@@ -44,14 +44,61 @@ archs = ["x86_64", "arm64"]
 [tool.cibuildwheel.linux]
 archs = ["auto", "aarch64"]
 
+[tool.black]
+line-length = 100
```

Comment:
I think we agreed on 88 ? (flake8 is set to 88 below, what's the default for isort ?)